### ISSUE TITLE
Global Simvars not changed

### ISSUE THREAD

**Issue by mixMugz** [flybywiresim/a32nx#5855]
### Aircraft Version

Development

### Build info

```json
{
  "built": "2021-09-19T13:44:38+00:00",
  "ref": "master",
  "sha": "80496bdd2b5281884df0ba379713b0cefbdce7fa",
  "actor": "2hwk",
  "event_name": "manual"
}
```

### Describe the bug

Parking brake not changing A: BRAKE PARKING INDICATOR or A:BRAKE PARKING PORITION.
Also engaging autopilot not changing A:AUTOPILOT MASTER

It's possible to addition use Aircraft Simvar to indicate state of these system?

### Expected behavior

Simulation SimVars must change depend of state

### Steps to reproduce

1. Load aircraft
2. Load SimWar Watcher
3. Change Autopilot/ParkBrake
4. Look simwars

**Comment by Benjozork** [comment]
The first is not possible due to technical limitations.

@aguther would know for the second - however you can use our LVars for it.

**Comment by mixMugz** [comment]
> The first is not possible due to technical limitations.

can you explain about limitation? All other aircraft (even custom Aerosoft/PMDG work with this variable) work with this simvar, why A32NX not work?

**Comment by sidnov** [comment]
We use brakes simulated by actual hydraulic systems, that cannot be simply controlled by a 0/1 brake system in the sim

**Comment by Benjozork** [comment]
We use a custom autobrake system and disabling the one included with MSFS disables this variable.

**Comment by mixMugz** [comment]
I understand. But it's possible to set sib parkbrake to 1 when all the conditions for this are met and parking brakes are actually set on the aircraft?

**Comment by Benjozork** [comment]
No. The simvar becomes locked when disabling MSFS' parking brake.

**Comment by aguther** [comment]
Same is for the AP, it‘s not possible to set it or at least I don‘t know a way how to do it. If you know a way let me know.

**Comment by 2hwk** [comment]
Reinvestigate after SU6

**Comment by derl30n** [comment]
Do we drop support for global sim vars in case of fully simulated custom systems? how to proceed with these issues?
